.code-samples.meilisearch.yaml

```diff
@@ -1061,3 +1061,14 @@ delete_documents_by_filter_1: |-
     --data-binary '{
       "filter": "genres = action OR genres = adventure"
     }'
+get_experimental_features_1: |-
+  curl \
+    -X GET 'http://localhost:7700/experimental-features/' \
+    -H 'Content-Type: application/json'  \
```

Comment:
Perhaps something has changed in the most recent versions, but I think we usually don't need to send content-type headers for GET requests.